### ISSUE TITLE
BA searches are not looking at mother's surname

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#1480]
Priority (1 3 2 5) 11

**Comment by edickens** [comment]
We have had this report through the transcribers group, but I have not verified it.  All the facts seem to be here.

I tried a search for..

John , natural son of Margar't Dixy , and reputed son of Willm. Nice Jun'r , bn 15/12/1730 , bap 03/1/1730/1 , in Bulmer, Essex.

I knew it was there because it was me who transcribed it.

He was known as John Dixey but I could not find the record under that name or similar. (He appears as such on Family Search).   Eventually by ticking the 'family member' box I got John Nice to appear, but only by looking at the record was the mother seen.

My point is that the search appears only to work on the father's surname. There must be thousands of records entered where two surnames or just the female one is entered. If the child is known by the mother's surname it is going to be missed.

This appears to me to be a major error in the Search routine and I think it should be addressed as a matter of urgency.

**Comment by Sherlock21** [comment]
i just did a search of one of my transcriptions ( in Prod) where the file has child's first name and mother's first name and surname ( in the right fields)this comes up in a search correctly with no need to use 'Family'.

so that bit of the story seems difficult to understand.
I don't have to hand any records where there are different surnames, But might play in T3 to see.

The one i used in the above search is: DEV/ Sidbury/ Maria/ HOLMYEAR/ 1813.

**Comment by Sherlock21** [comment]
Testing this out in T3, on an existing Baptism batch where there are different surnames for Male & Female parents, a search of child: James + mother's surname FISHER /1796/ Dewsbury - does not return the test record

**Comment by edickens** [comment]
Does it work with "Family members" switched on?

**Comment by SteveBiggs** [comment]
From my testing:
- If only a mother's surname is given, the record is found by a surname search
- If both father and mother surnames given, the child's surname is assumed to be that of the father so a search with only mother's does not return result
- If both father and mother surnames given, a search for JUST mother's surname with Family members ticked does find result, but if child's forename is also in search, no result is returned

**Comment by Captainkirkdawson** [comment]
This feature was never programmed it is scheduled for 1.7 as part of the roadmap

**Comment by stsccfr** [comment]
Same issue as #1211 in FreeUKGenProduct

**Comment by DeniseColbert** [comment]
@stsccfr to perform a test on a county that has been reprocessed to see if the emendation task is solving this issue

**Comment by stsccfr** [comment]
Yes, this is working now.  If there is no value for person_surname, and the search name matches either the father_surname or the mother_surname, then the record is found.  I will close this case and the related 1211 in FreeUKGenProduct
